pkgs/applications/networking/cluster/talosctl/default.nix

```diff
@@ -56,5 +35,8 @@ buildGoModule rec {
     homepage = "https://www.talos.dev/";
     license = licenses.mpl20;
     maintainers = with maintainers; [ flokli ];
+    # requires >= 10.14 SDK https://github.com/NixOS/nixpkgs/issues/101229
+    # Undefined symbols for architecture x86_64: "_SecTrustEvaluateWithError"
```

Comment:
So this is not a problem on aarch64-darwin?

Reply:
yes, aarch64 is fine.